sdk/storage/storage-blob/review/storage-blob.api.md

```diff
@@ -393,6 +396,7 @@ export interface BlobCopyFromURLHeaders {
     lastModified?: Date;
     requestId?: string;
     version?: string;
```

Comment:
:( It would've been better if we named this `serviceVersion` instead of just `version` . Too late now...